scripts/write_cmpd_to_frag_dict.py

```diff
@@ -0,0 +1,56 @@
+"""
+Build library of ligands from a dataset of holo crystal structures docked to a
+different dataset of apo structures.
+"""
```

Comment:
I think we should say here that the output is a YAML file and maybe talk about the format of the file in broad terms.